.github/workflows/code_size_comment.yml

```diff
@@ -0,0 +1,33 @@
+name: Code size comment
+
+on:
+  workflow_run:
+    workflows: [Check code size]
+    types: [completed]
+
+jobs:
+  comment:
+    runs-on: ubuntu-20.04
+    steps:
+    - name: Download code size report
+      uses: actions/download-artifact@v3
```

Comment:
If this fails, then we will need to use the github api instead as in the example from the docs.

```yaml
      - name: 'Download artifact'
        uses: actions/github-script@v6
        with:
          script: |
            const allArtifacts = await github.rest.actions.listWorkflowRunArtifacts({
               owner: context.repo.owner,
               repo: context.repo.repo,
               run_id: context.payload.workflow_run.id,
            });
            const matchArtifact = allArtifacts.data.artifacts.filter((artifact) => {
              return artifact.name == "code-size-report"
            })[0];
            const download = await github.rest.actions.downloadArtifact({
               owner: context.repo.owner,
               repo: context.repo.repo,
               artifact_id: matchArtifact.id,
               archive_format: 'zip',
            });
            const fs = require('fs');
            fs.writeFileSync(`${process.env.GITHUB_WORKSPACE}/code-size-report.zip`, Buffer.from(download.data));

      - name: 'Unzip artifact'
        run: unzip code-size-report.zip
```